env0/provider_test.go

```diff
@@ -74,12 +74,12 @@ func testMissingEnvVar(t *testing.T, envVars map[string]string, expectedKey stri
 		defer os.Setenv(key, "")
 	}
 
-	diags := Provider("TEST")().Validate(&terraform.ResourceConfig{})
+	diags := Provider("TEST")().Configure(context.Background(), &terraform.ResourceConfig{})
```

Comment:
Configure is at later stage that returns the error. (Because now schema passes).